mlops-multi-account-cdk/mlops-sm-project-template/seed_code/deploy_app/deploy_endpoint/deploy_endpoint_stack.py

```diff
@@ -254,3 +289,67 @@ def __init__(
         endpoint.add_depends_on(endpoint_config)
 
         self.endpoint = endpoint
+
+    def create_batch_inference_resources(self):
+        """
+        create a sagemaker pipeline that contains batch inference
+        """
+        prj_name = PROJECT_NAME.lower()
+        self.transform_bucket = f"{prj_name}-trnsfrm-{DEV_ACCOUNT}"
+        self.pipeline_name = f"{PROJECT_NAME}-transform"
+
+        # upload code asset to the default bucket
+        BASE_DIR = os.path.dirname(os.path.realpath(__file__))
+
+        i_bucket = s3.Bucket(
+            self,
+            id=self.transform_bucket,
+            bucket_name=Fn.sub(
+                self.transform_bucket.replace(DEV_ACCOUNT, "${AWS::AccountId}")
+            ),
+            versioned=False,
+            removal_policy=RemovalPolicy.DESTROY,
+            encryption=s3.BucketEncryption.S3_MANAGED           
+        )
+
+        source_scripts = s3_deployment.BucketDeployment(
+            self,
+            id=f"{prj_name}-source_scripts",
+            destination_bucket=i_bucket,
+            destination_key_prefix=f"{self.pipeline_name}/{self.timestamp}/source-scripts",
+            sources=[s3_deployment.Source.asset(path=f"{BASE_DIR}/source_scripts")],
```

Comment:
As per our discussion, the source scripts seem to be missing in this deploy repository. I'll test as soon as you add them :)

Reply:
Source scripts added